OJP/OJP_Trips.xsd

```diff
@@ -106,18 +107,54 @@
+			<xs:documentation>List of Access Modes.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:group ref="BaseTripPolicyGroup"/>
-			<xs:element name="ItModesToCover" type="IndividualModesEnumeration" minOccurs="0" maxOccurs="unbounded">
+			<xs:element name="AccessMode" type="ifopt:AccessModesEnumeration" minOccurs="0" maxOccurs="unbounded">
 				<xs:annotation>
-					<xs:documentation>For each mode in this list a separate monomodal trip shall be found - in addition to inter-modal solutions.</xs:documentation>
+					<xs:documentation>Access mode that should be considered.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 		</xs:sequence>
+	</xs:complexType>
+	<xs:group name="TripPolicyGroup">
+		<xs:annotation>
+			<xs:documentation>Policies that control the trip search behaviour.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:group ref="BaseTripPolicyGroup"/>
+			<xs:choice>
+				<xs:element name="MotorizedMainTravelMode" type="xs:boolean" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation>The MotorizedMainTravelMode defines whether the trip planner works in PT mode (or pure IT mode) or includes carTransportRail and the like. If set to true (e.g. because one travels by car, truck or motorcycle), then carTransportRail, ferry are used together with roads.</xs:documentation>
+					</xs:annotation>
+				</xs:element>
+				<xs:sequence>
+					<xs:element name="ItModesToCover" type="IndividualModesEnumeration" minOccurs="0" maxOccurs="unbounded">
+						<xs:annotation>
+							<xs:documentation>For each mode in this list a separate monomodal trip shall be found - in addition to inter-modal solutions.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="IncludeAlternativeOptions" type="xs:boolean" default="false" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>Whether alternative options should be presented as well. Mainly important for dominated journeys or in the case of ContinuousLegs the second-best route. Should be optimised for the user expectance (see. e.g https://theses.hal.science/tel-01848737). However, what the alternative options are may vary widely depending on the optimisation methods and filters.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="IncludeRestrictedLines" type="xs:boolean" minOccurs="0" default="false">
+						<xs:annotation>
+							<xs:documentation>There might exist lines that have special restrictions and are not generally available to the public. E.g. school buses, company shuttles. dragLifts need to have an AccessMode ski. If this flag is set, then existing restricted lines are considered by the router. The results are marked as restricted in the ServiceSection. The restriction can be seen from: ACCESS MODE or PrivatMode. The usage could also detailed with Attribute elements.</xs:documentation>
```

Comment:
```suggestion
							<xs:documentation>There might exist lines that have special restrictions and are not generally available to the public. E.g. school buses, company shuttles. dragLifts need to have an AccessMode ski. Lines with ACCESS MODE bicycle will be included as well. If this flag is set, then existing restricted lines are considered by the router. The results are marked as restricted in the ServiceSection. The restriction can be seen from: ACCESS MODE or PrivatMode. The usage could also be detailed with Attribute elements.</xs:documentation>
```

Reply:
Please have a look at the bicycle proposal; if it should be the other way round, I think it would be good to state that, too.

Reply:
@trurlurl  bicycles are available by default (as you can use them on most publi transport)

Reply:
Im Fall, den du erwähnst («bicycles are available by default») ist das bicycle nicht access mode, man braucht es ja nicht um mitfahren zu dürfen. Beim Beispiel Goppenstein-Kandersteg ist es (oder könnte es zumindest sein) ein access mode, neben car und motorcycle.

Mein Punkt hier wäre: ich fände es gut, klar zu definieren, wie das bicycle behandelt werden soll, damit man als requester weiss, ob bicycle als access mode mitgeteilt wird oder nicht. 

Für mich ist die Sache eigentlich auch bei car, motorcycle, truck nicht so ganz eindeutig. Nach der annotation und der Tatsache, dass es bereits MotorisedMainTravelMode gibt, würde ich eher nicht damit rechnen, dass car- oder motorcycle-Strecken freigeschaltet werden. Aber der Name «IncludeRestrictedLines»  alleine würde vermuten lassen, dass jegliche restricted lines berücksichtigt werden. 

Ich glaube, dass es die bessere Lösung wäre, wenn bicycle berücksichtig wird bei IncludeRestrictedLines, und nicht nur wenn BikeTransport angewählt ist. Z.B. bei dragLifts mit access mode bike, da kommt einem erst durch das Angebot die Idee, ein Velo mitzunehmen.

Wenn der Annotationstext angepasst wird, sollte auch die annotation zu IncludeRestrictedLines in OJP_StopEvents.xsd angepasst werden.

@trurlurl will use your suggest. pls do the change directly in all places, where it needs to be done.